README.md

```diff
@@ -64,6 +64,10 @@ Options:
           The User-Agent for Reddit API requests [env: SHREDDIT_USER_AGENT=ShredditRustClient] [default: ShredditRustClient]
       --gdpr-export-dir <GDPR_EXPORT_DIR>
           The path of the directory of the unzipped GDPR export data. If set, `shreddit` will use the GDPR export folder instead of Reddit's APIs for discovering your data [env: SHREDDIT_GDPR_EXPORT_DIR=/home/you/Downloads/export_yourusername_20230101]
+      --prevent_comment_deletion
```

Comment:
Could you call this `--edit-only` and make this never delete anything (regardless of `ThingType`? I think that would make it simpler for users.

Reply:
That should be easy enough. I can put the check into the overall shred impl in the mod.Rs file, so that delete() is never called.

Reply:
Just need to update this README example and we should be good to go!

Reply:
Just committed a change to the readme